packages/vm/lib/evm/types.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Logs that the contract emitted.
```

Comment:
plural -> singular for clarity

```suggestion
 * Log that the contract emitted.
```